### DISCUSSION TITLE
Ignore "mia-platform.eu/version" annotation when merging branches

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#71] [Ideas]
When merging branches saved with two different versions of Mia-Platform console, the annotation `mia-platform.eu/version` gets updated in all the microservices of my project.

While merging the two configurations, for a developer prospective this change is just noise. I think that it should be better to ignore this change in the merge modal.

**Comment by davidebianchi** [comment]
This could be an interesting improvement, from the users point of view this data could be removed as will be updated by the backend at each save of configuration. @fredmaggiowski @albertotessarotto

**Reply by albertotessarotto** [reply]
I agree, this info can be ignored by users during merge

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
